modules/ROOT/pages/project-structure.adoc

```diff
@@ -1,89 +1,142 @@
-To give full freedom to the professional developers, they can be created with other tools than Bonita and added to the project when needed. 
+Note that not all elements are required and they are alternative ways of doing. However, using the provided elements enables to use the full potential of the Bonita platform.
 
-== Main elements
-
-[cols="1,1,1,1"]
+=== Process Elements
+[cols="1,1,1,1,1"]
 |===
-|Element |Create/Provided | Tool | Alternative
+|Element |Description |Create/Provided | Tool | Alternative
 
 |Processes
+|BPMN diagram that describes your business processes
 |Create
 |Studio
 |--
 
 |Forms
+|Web interfaces used for human interactions with your processes
 |Create
 |UI Designer
 |Use BPM REST API
 
-|Business Data Model
+|Actor filter
+|Filters users that can access a human task in a process
+|A couple are provided in the Marketplace and you can create custom actor filters
+|Studio or SDK (custom connectors)
+|-
+
+|Connectors
+|Web interfaces used for human interactions with your processes
+|A dozen connectors are provided in the Marketplace and you can create custom connectors
+|Studio or SDK (custom connectors)
+|-
+
+
+|=== 
+
+=== Data
+[cols="1,1,1,1,1"]
+|===
+|Element |Description |Create/Provided | Tool | Alternative
+
+|Business Data Model(BDM)
 |Create
+|Structure of the business data to be used with your processes
 |Studio
 |External data base w/ provided connector
 
+|=== 
+
+=== Identity Management
+[cols="1,1,1,1,1"]
+|===
+|Element |Description |Create/Provided | Tool | Alternative
+
 |Organization (for "Actors" and "Profiles")
+|The definition of the user base that will use the processes (includes users, roles and groups)
 |Example provided/Create
 |Studio
 |--
+|===
+
+=== Living Applications
+[cols="1,1,1,1,1"]
+|===
+|Element |Description |Create/Provided | Tool | Alternative
 
-|Application decriptor
+|Application descriptor
+|Defines the *living applications*: group of pages with a specific them for a given user profile
 |Create
 |Studio
 |xml editor
 
 |Application pages
+|Web pages used within your applications
 |Create
 |UI Designer
 |Front-end framework
 
 |Layout
+|Used to define the main frame of an application.
 |Provided
 |UI Designer
 |Front-end framework
 
 |Theme
-|Provided
+|Defines the look and feel of your application
+|A default theme is provided and you can create custom theme
 |--
 |--
 
 |=== 
 
-For an Automation Project with no application, the last 4 elements are not needed.
+For a Bonita Automation project with no application, the last 4 elements are not needed.
 
-== How main elements fit together
+== Extensions
+Bonita Automation project can be extended by creating custom components. These are called extensions.
 
-This image shows what an automation project looks like when it is about creating a Living Application:
+Some extensions are used to add elements to those provided while other are used to enable more possibilities in the platform.
```

Comment:
I do not understand the sentence 'Some extensions are used to add elements to those provided', what are you trying to say? 

```suggestion
Some extensions are used to add elements to those provided while others are used to enforce the Bonita platform possibilities.
```